### PR TITLE
Fix use of key pos API, and point to first tensor key in command def

### PR DESCRIPTION
**Pull request by lantiga**
Addresses #160 
/cc @MeirShpilraien

### CODE REVIEW

File: src/redisai.c [hunk 1]
```diff
@@ -688,14 +688,22 @@ int RedisAI_ModelRun_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv,
   // be picked up on the next round. We also need to signal when it's time to dispose
   // of the old model.
   // The key is having a single thread looping for execution
+  if (argc < 3) return RedisModule_WrongArity(ctx);
+
   if (RedisModule_IsKeysPositionRequest(ctx)) {
     RedisModule_KeyAtPos(ctx, 1);
+    for (int i=1; i<argc; i++) {
```
Comment (MeirShpilraien) [inline src/redisai.c]: shouldn't you start from 2?

File: src/redisai.c [hunk 2]
```diff
@@ -848,14 +856,22 @@ int RedisAI_StartRunThread() {
 
 // script key, fnname, INPUTS, key1, key2 ... OUTPUTS, key1, key2 ...
 int RedisAI_ScriptRun_RedisCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
+  if (argc < 4) return RedisModule_WrongArity(ctx);
+
   if (RedisModule_IsKeysPositionRequest(ctx)) {
     RedisModule_KeyAtPos(ctx, 1);
+    for (int i=2; i<argc; i++) {
```
Comment (MeirShpilraien) [inline src/redisai.c]: shouldn't you start from 3?

File: src/redisai.c (hunk 1 above)
Comment (lantiga) [reply src/redisai.c]: Damn

**Comment by lantiga** [comment]
Fixed, good catch :-)

**Pull request merged**